src/test/java/io/reactivex/rxjava3/observers/SerializedObserverTest.java

```diff
@@ -340,11 +340,11 @@ public void onNext(String t) {
      *
      * When using SynchronizedSubscriber we get this output:
      *
-     * p1: 18 p2: 68 => should be close to each other unless we have thread starvation
+     * {@code p1: 18 p2: 68 => should be close to each other unless we have thread starvation}
      *
      * When using SerializedObserver we get:
      *
-     * p1: 1 p2: 2445261 => should be close to each other unless we have thread starvation
+     * {@code p1: 1 p2: 2445261 => should be close to each other unless we have thread starvation}
```

Comment:
No reason to include "should be close to each other unless we have thread starvation" in the code block.